base/ca/src/main/java/com/netscape/cms/servlet/request/ProcessCertReq.java

```diff
@@ -1518,7 +1515,9 @@ else if (grantDRMAgent)
 
                 throw new ECMSGWException(CMS.getUserMessage("CMS_GW_UNAUTHORIZED_CREATE_GROUP", obj[0]));
             }
-        }
+        } else {
+         return 0;
+      }
```

Comment:
Yeah the original indentation was weird and it survived the transformation. I tried to avoid whitespace changes but as you brought it up I will fix it.